src/utils/sentry.ts

```diff
@@ -73,7 +73,8 @@ function initSentry() {
     maxBreadcrumbs: 50,
     attachStacktrace: true,
     integrations: [new BrowserTracing()],
-    tracesSampleRate: 0.007,
+    tracesSampleRate: 0.2,
+    debug: true,
```

Comment:
We usually use local storage flags to enable dbeugging modes.

We have these functions: https://github.com/RedHatInsights/insights-chrome/blob/dd4b078586c5981781eca0636b4759050f2b7492/src/utils/debugFunctions.ts

They are available on `window.insights.chrome.enable`, so you can turn on/off the debugging when required.

Reply:
Oh, cool. PTAL at the updated commit.